hack/generator/pkg/astmodel/testdata/NastyTest.golden

```diff
@@ -31,8 +31,8 @@ func (person Person) ConvertFrom(source *vHub.Person) error {
 	return nil
 }
 
-// ConvertTo populates the provided destination Person from our Person
-func (person Person) ConvertTo(destination *vHub.Person) error {
+// ConvertToVNext populates the provided destination Person from our Person
+func (person Person) ConvertToVNext(destination *vNext.Person) error {
```

Comment:
minor: I generally favor full words in methods. Would `ConvertToNextVersion` and `ConvertFromNextVersion` (or _possibly_ `ConvertToHigherVersion` and `ConvertFromHigherVersion`, assuming that my assumption that your "vnext" is always a "bigger" version (orders after)) be clearer?

Reply:
The `VNext` in the method name comes from the `vNext` namespace of the parameter. 

In actual use, the namespace will be something like `v1api20180101` so the method name will be `ConvertToV1api20180101` which is really explicit.

Reply:
Totally missed that